concepts/central-resource-sync/README.md

```diff
@@ -0,0 +1,135 @@
+
+## General Information
+
+### Stakeholders and Roles
+
+| Role                   | Name                                                   |
+|------------------------|--------------------------------------------------------|
+| Author(s)              | Xin Ruan (@ruanxin)                                    |
+| Architect              | Xin Ruan (@ruanxin)                                    |
+| Product Owner          | Jan Medrek (@janmedrek), Piotr Bochyński (@pbochynski) |
+| Lifecycle Manager Team | @kyma-project/jellyfish                                |
+
+## Design
+
+### Key Requirements and Design Goals
+
+The Module Synchronization design aims to meet the following key requirements and design goals:
+
+- Efficient Synchronization: A primary objective is to ensure the efficient synchronization of module resources between the KCP cluster and the SKR cluster. This involves timely and accurate updates to maintain the desired state of module resources across clusters.
+- Responsibility Separation: LM takes charge of managing the synchronization of module resources, while Module Manager, owned by individual module teams, is responsible for providing the necessary resources.
+- Minimized Central Components access SKR cluster: To optimize efficiency and security, this design avoids introducing additional central components with direct connections to remote clusters. By reducing the number of connections and limiting components with cluster-admin access to the SKR cluster, the system mitigates security risks associated with privileged access.
+- Real-time Reflection of Changes: A critical aspect of the design is to ensure that any changes to client resources, including Module CR and Kyma CR, are reflected in module resources instantaneously. Real-time synchronization is essential to maintain system integrity and responsiveness.
+
+### Context
+
+This architecture design offers a unified solution for synchronization of module resources between the KCP cluster and the SKR cluster.
+
+### Major Building Blocks
+
+![](assets/sync-resource.svg)
+
+#### Lifecycle Manager 
+The existing Lifecycle Manager plays a central role in managing the synchronization of module resources. It orchestrates the deployment and synchronization of module resources into the remote SKR Cluster, acting as the sole gateway to access this cluster within the specified setup.
+
+#### Module Manager 
+Module Managers are components created and maintained by individual module teams. Their purpose is to offer SyncResource CRs based on configuration data received from the SKR Cluster.
+
+#### SKR Watcher
+SKR Watcher is an existing admission webhook service deployed by LM in the SKR Cluster. Its main role is to receive validation admission requests from Module CRs and Kyma CRs and send the requests to LM. This enables LM to enqueue and reconcile related Kyma CRs immediately without waiting for scheduled time intervals.
```

Comment:
You cannot receive requests from a CR, please formulate better. Maybe:
Its main role is to detect modifications to Module CRs and Kyma CRs by using validation admission webhooks and inform the central LM about these changes in a timely manner